MapboxNavigation/UILabel.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+protocol Abridgeable: class {
+    var unabridgedText: String? { get set }
+    typealias AvailableBoundsHandler = () -> (CGRect)
+    var availableBounds: AvailableBoundsHandler! { get set }
+}
+
+extension Abridgeable where Self: UILabel {
+    func set(unabridgedText: String?) {
```

Comment:
That's a good improvement but I was hoping for an even better ;)
Overriding unabridgedText's didSet or observing changes and update super.text accordingly would be optimal but I couldn't figure out a way to achieve that in an extension, that's why I detoured through generics. However, `setUnabridgedText(_:)` will also do the job. 👍